### ISSUE TITLE
[Feature Request] Telegram Integration

### ISSUE THREAD

**Issue by davidnewhall** [Notifiarr/website#22]
FR: Integrate Telegram Notifications

I do not personally need this, but I know plenty of folks will use it, and it will draw more users into the product.

**Comment by davidnewhall** [comment]
For inspiration and deprecation: https://github.com/toddrob99/searcharr

**Comment by VozDeOuro** [comment]
Im very interested in this. Can we expect this anytime soon?

**Comment by austinwbest** [comment]
> Im very interested in this. Can we expect this anytime soon?

I have some mock up code and things from very basic testing but nothing that is close to production ready. The way telegram works compared to discord is very different in workflow/implementation and it left me with a long list of questions to figure out before i could even start trying to actually implement it. 

Basically i have to figure out which features are currently in the workflow with discord that are just not supported/possible with telegram before i can even begin as i have to handle those accordingly.

Here are some for example

- how to get a list of all groups the bot is in? need this to get the channel list
- how to get a list of all channels in a group? need this to assign triggers
- how to get a list of all users in a group? need this for permissions
- how to add new channels to a group? not sure if this is needed
- how to add a new message in a specific channel in a group?
- how to update message in a specific channel in a group?
- how to delete a message in a specific channel in a group?
- on the site, add a telegram specific apikey?
- on the site, just add the bot token?

**Comment by iceman73** [comment]
Any way we could just get telegram notifications for now?

**Comment by bakerboy448** [comment]
> Any way we could just get telegram notifications for now?

no. the site would need to be completely modified to add support for telegram

**Comment by austinwbest** [comment]
> Any way we could just get telegram notifications for now?

That list of questions has not a single response so even if i wanted to do it right now, that is a lot of things standing in the way

**Comment by davidnewhall** [comment]
https://github.com/php-telegram-bot/core
